components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go

```diff
@@ -20,6 +20,9 @@ const (
 
 	// Finalizer for deleting Knative Subscriptions
 	finalizerName = "subscription.finalizers.kyma-project.io"
+	subscriptionSourceID = "kyma-source-id"
```

Comment:
Could you add an empty line after "finalizerName" and before "subscriptionSourceID"